### PR TITLE
May Update 2023

### PR DESCRIPTION
**Pull request by somtochiama**
https://deploy-preview-1513--fluxcd.netlify.app/blog/2023/06/may-2023-update/

### CODE REVIEW

File: templates/monthly-update.md [hunk 1]
```diff
@@ -130,13 +128,6 @@ things. Give us a shout-out and we will link it from this section! ✍
 
 #### Flux Adopters shout-out
 
-<!-- 
-
-Please consult https://github.com/fluxcd/website/commits/main/data/adopters
-for this.
-
--->
-
```
Comment (dholbach) [inline templates/monthly-update.md]: I think it's good to leave a comment here so folks have a reminder where to look for this kind of information.

**Review by mewzherder — APPROVED** [review]
lgtm

File: content/en/blog/2023-06-01-may-update/index.md [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+- [Blablacar](https://www.blablacar.com/): a long distance carpooling platform that connects drivers with empty seats and passengers to share travel costs.
+
+Thanks to Horacio Granillo ([@hgranillo](https://github.com/hgranillo)),
+Peter König ([@konigpeter](https://github.com/konigpeter)), and Julien Haumont
+([@jhaumont](https://github.com/jhaumont)) for taking the time to make these
+additions to the Flux adopters list!
+
+*If you have not already done so, use the instructions here or give us a ping
+and we will help to add you. Not only is it great for us to get to know and
+welcome you to our community. It also gives the team a big boost in morale
```
Comment (aryan9600) [inline content/en/blog/2023-06-01-may-update/index.md:109]:
```suggestion
welcome you to our community, it also gives the team a big boost in morale
```

File: content/en/blog/2023-06-01-may-update/index.md [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+
+### Flux members, contributors, and maintainers!
+
+#### Priyanka Ravi joins as Flux Project Member
+
+We are very happy that Priyanka "Pinky" Ravi [joined us as a Flux
+Project Member](https://github.com/fluxcd/community/issues/293).
+
+Over the past years, Pinky spoke at conferences, meetups and elsewhere.
+Demoing Flux, discussing use-cases and discussing what's new. If you
```
Comment (aryan9600) [inline content/en/blog/2023-06-01-may-update/index.md:120]:
```suggestion
Over the past years, Pinky spoke at conferences, meetups and elsewhere, 
demoing Flux, discussing use-cases and discussing what's new. If you
```

File: content/en/blog/2023-06-01-may-update/index.md [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+her to the team.
+
+#### Sanskar Jaiswal becomes a Core Maintainer
+
+Sanskar has been making major code contributions to Flux
+for a while and is already a Flagger maintainer. He has
+been instrumental in getting the improving the git implementation
+in Flux and a host of other features.
+
+Thanks for all your contributions to Flux! This is well-deserved
```
Comment (aryan9600) [inline content/en/blog/2023-06-01-may-update/index.md:143]:
```suggestion
Thanks for all your contributions to Flux! This is well-deserved.
```

File: content/en/blog/2023-06-01-may-update/index.md [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+
+#### Mehak Saeed selected for Flux's Season of Docs
+
+We are excited to welcome Mehak Saheed who would be working to
+improve Flux's documentation during this year's Google Season
+of Docs. Mehak is a technical writer with over six years of experience
+and has worked on documentation for projects such as [cert-manager]
+(https://cert-manager.io/docs/) and [Unfurl](https://cert-manager.io/docs/).
```
Comment (aryan9600) [inline content/en/blog/2023-06-01-may-update/index.md:151]: wrong link for Unfurl

**Review by aryan9600 — CHANGES_REQUESTED** [review]

File: content/en/blog/2023-06-01-may-update/index.md [hunk 6]
```diff
@@ -0,0 +1,271 @@
...
+
+### Three more Flux release candidates! many improvements - please test
+
+On our path to GA, we released v2.0.0-rc5, the fifth release candidate for
+the 2.0.0 release. It includes many fixes, so you are very much encouraged to
+upgrade to this latest version - even though it carries "RC" in its version
+number, it is the most stable Flux release to date. Users are advised to
+upgrade from v0.41 and older versions to v2.0.0-rc.5 as soon as possible.
+
+#### Fixes and improvements
```
[the remark this answers is not in the document]
Comment (somtochiama) [reply content/en/blog/2023-06-01-may-update/index.md:41]: I don't think we need to add everything across all the RC but I will add the support for cosign and helm

File: content/en/blog/2023-06-01-may-update/index.md [hunk 7]
```diff
@@ -60,31 +59,35 @@ upgrade from v0.41 and older versions to v2.0.0-rc.5 as soon as possible.
...
   sorting them by key (helm-controller)
+- Update cosign to v2 (source-controller)
+- Support for Helm 3.12.0 and Kustomize v5.0.3.
 
 To upgrade from v0.x to v2.0.0-rc.5, please see
 [the procedure documented in RC.1.](https://github.com/fluxcd/flux2/releases/tag/v2.0.0-rc.1).
 
 :warning: Note that Kubernetes 1.27.0 contains a regression bug that affects
-Flux, it is recommended to upgrade Kubernetes to 1.27.1 or newer.
+Flux, it is recommended to upgrade Kubernetes to 1.27.1 or newer. The upgarde to
```
Comment (dholbach) [inline content/en/blog/2023-06-01-may-update/index.md:69]:
```suggestion
Flux, it is recommended to upgrade Kubernetes to 1.27.1 or newer. The upgrade to
```

File: content/en/blog/2023-06-01-may-update/index.md [hunk 8]
```diff
@@ -0,0 +1,274 @@
...
+- [VS Code+Flux: Dev-Driven Automated Deployments Like a Cloud Native Pro (Even if You’re a Beginner)](https://youtu.be/biC7X33o9eI) - Flux ecosystem contributor, Juozas Gaigalas, Weaveworks
+- [Level Up Your Deployments: Automate with Terraform + Flux](https://youtu.be/R4rKr4jbvr8) - Flux contributor, Priyanka "Pinky" Ravi, Weaveworks
+- [Platform Engineering Done Right: Safe, Secure, & Scalable Multi-Tenant GitOps](https://youtu.be/aScxRi6sjrk) - Flux ecosystem contributor, Juozas Gaigalas, Weaveworks
+- [Exotic Runtime Targets: Ruby and Wasm on Kubernetes and GitOps Delivery Pipelines (40-min version)](https://youtu.be/EsAuJmHYWgI) - Flux maintainer, Kingdon Barrett, Weaveworks
+- [Lightning Talk: GitOps Sustainability with Flux and arm64 (5-min version)](https://youtu.be/vBQ3wN1c9xU) - Flux contributor, Tamao Nakahara, Weaveworks
+- [Community Diversity and Inclusion as Business Metric (and Not Just a Feel-Good Tactic)](https://youtu.be/A-su3Rb7UC8) - Flux contributor, Tamao Nakahara, Weaveworks
+
+{{< gallery match="images/*" sortOrder="desc" rowHeight="150" margins="5"
```
Comment (dholbach) [inline content/en/blog/2023-06-01-may-update/index.md:215]: Maybe increase the size of the photos  somewhat?

File: content/en/blog/2023-06-01-may-update/index.md [hunk 9]
```diff
@@ -0,0 +1,274 @@
...
+
+May was packed with exciting stories from Flux users, newly updated
+Flux adopters, contributors, contributions and a new GA release candidate!
+Also, don’t miss future Flux Bug Scrubs using ChatGPT.
+
+## Flux technology things to know
+
+### Three more Flux release candidates! many improvements - please test
+
+On our path to GA, we released v2.0.0-rc5, the fifth release candidate for
```
Comment (makkes) [inline content/en/blog/2023-06-01-may-update/index.md:34]:
```suggestion
On our path to GA, we released [v2.0.0-rc5](https://github.com/fluxcd/flux2/releases/tag/v2.0.0-rc.5), the fifth release candidate for
```

File: content/en/blog/2023-06-01-may-update/index.md [hunk 10]
```diff
@@ -0,0 +1,274 @@
...
+Please see the [Changelog](https://github.com/fluxcd/flagger/blob/main/CHANGELOG.md#1310)
+on how to upgrade.
+
+## News from Flux users & the Community!
+
+### Newly posted Flux adopters!
+
+{{< gallery match="logos/*" sortOrder="desc" rowHeight="150" margins="5"
+            thumbnailResizeOptions="600x600 q90 Lanczos"
+            previewType="blur" embedPreview=true lastRow="nojustify" >}}
```
Comment (makkes) [inline content/en/blog/2023-06-01-may-update/index.md:98]: The rendered order of logos in here doesn't match the list below but I don't know how to fix that. If we can't find a way to change it, let's change the order of the list below to match the logos here.

File: content/en/blog/2023-06-01-may-update/index.md [hunk 11]
```diff
@@ -0,0 +1,274 @@
...
+Peter König ([@konigpeter](https://github.com/konigpeter)), and Julien Haumont
+([@jhaumont](https://github.com/jhaumont)) for taking the time to make these
+additions to the Flux adopters list!
+
+*If you have not already done so, [use the instructions here](/adopters/) or give us a ping
+and we will help to add you. Not only is it great for us to get to know and
+welcome you to our community. It also gives the team a big boost in morale
+to know where in the world Flux is used everywhere*
+
+### Flux members, contributors, and maintainers!
```
Comment (makkes) [inline content/en/blog/2023-06-01-may-update/index.md:115]: Love the shoutouts here!!

**Review by makkes — COMMENTED** [review]
Overall great post. Thanks @somtochiama. Just a couple of nits really.

**Review by makkes — APPROVED** [review]

File: content/en/blog/2023-06-01-may-update/index.md [hunk 12]
```diff
@@ -0,0 +1,275 @@
+---
+author: somtochiama
+date: 2023-05-10 7:30:00+00:00
```
Comment (hiddeco) [inline content/en/blog/2023-06-01-may-update/index.md:3]: This should be today, I think?

**Review by hiddeco — APPROVED** [review]

**Pull request merged**